### PR TITLE
Ignore deprecated release tag from manifests

### PR DESCRIPTION
**Pull request by jsoriano**
Stop showing information about the release tag of packages, to be deprecated in https://github.com/elastic/package-spec/pull/256.

Use the `prerelease` parameter when searching for packages, related to https://github.com/elastic/package-registry/pull/785.

Part of https://github.com/elastic/package-spec/issues/225

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -145,3 +145,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
+
+replace github.com/elastic/package-spec v1.3.0 => github.com/jsoriano/package-spec v0.0.0-20220112095620-98e0d531845d
```
Comment (jsoriano) [inline go.mod:149]: This change shouldn't be included.

Comment (mtojek) [reply go.mod:149]: I understand that this PR is blocked on the other one: https://github.com/elastic/package-spec/pull/256?

File: internal/registry/revisions.go [hunk 2]
```diff
@@ -19,9 +19,12 @@ import (
 
 // SearchOptions specify the query parameters without the package name for the search API
 type SearchOptions struct {
-	Experimental  bool   `url:"experimental"`
+	Prerelease    bool   `url:"prerelease"`
```
Comment (mtojek) [inline internal/registry/revisions.go:22]: ... oh, and on the EPR release.

**Review by mtojek — COMMENTED** [review]
Code changes LGTM, just left questions about dependent issues.

File: internal/registry/revisions.go (hunk 2 above)
Comment (jsoriano) [reply internal/registry/revisions.go:22]: Yes, I think this will be the last piece to merge.

**Review by mtojek — APPROVED** [review]
Approved, but considering the order of other pull requests :)

**Comment by jsoriano** [comment]
Testing with integrations repo in https://github.com/elastic/integrations/pull/2651

**Comment by jsoriano** [comment]
Added changes to implement in `elastic-package status` the same conventions for release labels than the ones decided for Kibana (https://github.com/elastic/kibana/issues/122973).

@mtojek could you please take another look?

**Review by mtojek — APPROVED** [review]
LGTM

**Pull request merged**